helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/model/AssignableNode.java

```diff
@@ -156,12 +153,12 @@ void release(AssignableReplica assignableReplica) throws IllegalArgumentExceptio
         .forEach(entry -> updateCapacityAndUtilization(entry.getKey(), -1 * entry.getValue()));
   }
 
-  public Map<String, Set<String>> getCurrentAssignmentsMap() {
+  public Map<String, Map<String, String>> getCurrentAssignmentsMap() {
```

Comment:
Could we please annotate the public methods, and especially makes sure the return values are marked with what the strings in the map stand for?

Reply:
I agree more comments would be helpful here. About annotation, could you please specify? Example would be helpful. Not sure if Helix has such a use case that I am not aware of?